src/Reflection/DocBlock/Tag/ParamTag.php

```diff
@@ -9,36 +9,27 @@
 
 class ParamTag implements TagInterface, PhpDocTypedTagInterface
 {
-    /**
-     * @var string[]
-     * @psalm-return list<string>
-     */
+    /** @var list<string> */
     protected $types = [];
 
-    /** @var string */
+    /** @var string|null */
     protected $variableName;
 
-    /** @var string */
+    /** @var string|null */
     protected $description;
 
-    /**
-     * @return string
-     */
+    /** @return 'param' */
     public function getName()
     {
         return 'param';
     }
 
-    /**
-     * Initializer
-     *
-     * @param  string $tagDocBlockLine
-     */
-    public function initialize($tagDocBlockLine)
+    /** {@inheritDoc} */
```

Comment:
Since this should be importing the whole docBlock, I would make it explicit https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc-tags.md#41-making-inheritance-explicit-using-the-inheritdoc-tag.

Reply:
dd68158de612598d53a7f3dcc96c27c5b3816dbf